include/TrackContentWidget.h

```diff
@@ -51,16 +51,22 @@ class TrackContentWidget : public QWidget, public JournallingObject
 	// qproperties for track background gradients
 	Q_PROPERTY( QBrush darkerColor READ darkerColor WRITE setDarkerColor )
 	Q_PROPERTY( QBrush lighterColor READ lighterColor WRITE setLighterColor )
-	Q_PROPERTY( QBrush gridColor READ gridColor WRITE setGridColor )
+	Q_PROPERTY( QBrush coarseGridColor READ coarseGridColor WRITE setCoarseGridColor )
```

Comment:
I hate to be that guy because I usually loathe these comments in reviews of my own code, but can you please unnecessary whitespace between the parentheses? The "rule" is that code that is added should not use these whitespace anymore.

Edit: Please check the diff and remove it everywhere in the new code.

Reply:
Haha, yeah I can do that.

Should I remove the whitespaces for the whole file for consistency's sake, or just the lines that I'm adding?

Reply:
Just the lines that you touch.

Reply:
There are still some places where whitespace shows up but for me that would be okay. It blends with the existing code and to me functionality is more important than code style (as long as it is not complete chaos).